_scripts/l10n/bundle.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+# Walks the l10n folder structure and combines ftl files into single master file.
```

Comment:
tiny nit: a couple of years ago we moved towards not referencing `master` where possible, maybe change this to "a single file" or "a single main file"? Same for the comment in `l10n/prime`.